File: Examples/CSharp/ProgrammersGuide/Working-with-Window-Elements/AddSupportOfVisualAids.cs
using Aspose.Diagram;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace CSharp.ProgrammersGuide.Working_with_Window_Elements
{
    public class AddSupportOfVisualAids
    {
        public static void Run() 
        {
            // ExStart:AddSupportOfVisualAids
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_WindowElements();

            // load source Visio diagram
            Diagram diagram = new Diagram(dataDir + "Drawing1.vsdx");

            // get window object by index
            Window window = diagram.Windows[0];
            // check dynamic grid option
            window.DynamicGridEnabled = BOOL.True;
            // check connection points option
            window.ShowConnectionPoints = BOOL.True;
            
            // save visio drawing
            diagram.Save(dataDir + "AddSupportOfVisualAids_Out.vsdx", SaveFileFormat.VSDX);
            // ExEnd:AddSupportOfVisualAids
        }
    }
}

File: Examples/VisualBasic/Working-with-Window-Elements/AddSupportOfVisualAids.vb
Imports Aspose.Diagram
Imports System
Public Class AddSupportOfVisualAids
    Public Shared Sub Run()
        ' ExStart:AddSupportOfVisualAids
        ' The path to the documents directory.
        Dim dataDir As String = RunExamples.GetDataDir_WindowElements()

        ' Load source Visio diagram
        Dim diagram As New Diagram(dataDir & Convert.ToString("Drawing1.vsdx"))

        ' Get window object by index
        Dim window As Window = diagram.Windows(0)
        ' Check dynamic grid option
        window.DynamicGridEnabled = BOOL.True

        ' Check connection points option
        window.ShowConnectionPoints = BOOL.True

        ' Save visio drawing
        diagram.Save(dataDir & Convert.ToString("AddSupportOfVisualAids_out.vsdx"), SaveFileFormat.VSDX)
        ' ExEnd:AddSupportOfVisualAids
    End Sub
End Class
